.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+const benniConfig = require('@baltpeter/prettier-config');
+
+module.exports = {
+    ...benniConfig,
+    plugins: [...(benniConfig.plugins ?? []), 'prettier-plugin-go-template'],
```

Comment:
There's a Prettier plugin for Go templates? :o 

We don't need this config merging, I definitely want that in my default config.

Reply:
Shall I create a PR?

Reply:
Sure.

Reply:
https://github.com/baltpeter/prettier-config/pull/1